README.md

```diff
@@ -197,7 +198,8 @@ It is recommended to use `polyglossia` (not `babel`).
 Note: there are other translations of HPMOR; here are listed only
 translations of this edition.
 
-* [French](https://github.com/yeKcim/hpmor) (in progress)
+* [French](https://github.com/yeKcim/hpmor) (complete)
+* [German](https://github.com/entorb/hpmor-de) (volume 1 (ch1-21) finished)
```

Comment:
Why not spell out "chapters 1–21"? Also wrap line 202.

Reply:
done